src/UglyToad.PdfPig/Content/Page.cs

```diff
@@ -193,6 +194,42 @@ public IEnumerable<Annotation> GetAnnotations()
             {
                 return annotationProvider.GetAnnotations();
             }
+
+            /// <summary>
+            /// Gets any optional content on the page.
+            /// <para>Does not handle XObjects and annotations for the time being.</para>
+            /// </summary>
+            public IDictionary<string, IReadOnlyList<OptionalContentGroupElement>> GetOptionalContents()
```

Comment:
I'd prefer `IReadOnlyDictionary<string, IReadOnlyList<OptionalContentGroupElement>>` since it is explicit that mutating the return type will not change anything on the `page` object